pyproject.toml

```diff
@@ -33,7 +33,6 @@ dependencies = [
     "ase>=3.19.0",
     "flatdict>=4.0.1",
     "hyperspy>=1.7.4",
-    "ifes_apt_tc_data_modeling<0.2,>=0.0.9",
```

Comment:
Is this the only dependency which was _only_ used by the apm reader?
All of the specific dependencies can be removed here now and should be added to `pyproject.toml` of the apm reader.

Reply:
Thank you @domna, yeah this also my understanding. Out of the top of my head this was indeed the only specifically-apm-used deps. If I find in during the pluginization there is one more I will remove them also.
Right now all other deps are mainly required by the em readers but with pynxtools-em also many of these will be removed